forte/evaluation/ner_evaluator.py

```diff
@@ -11,53 +11,85 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+"""Evaluator for Conll03 NER tag."""
 import os
-from pathlib import Path
 from typing import Dict
+from pathlib import Path
 
-from forte.data.data_pack import DataPack
-from forte.datasets.conll import conll_utils
+from ft.onto.base_ontology import Sentence
+from forte.common.configuration import Config
+from forte.common.resources import Resources
+from forte.data.base_pack import PackType
 from forte.evaluation.base import Evaluator
-from forte.processors.ner_predictor import CoNLLNERPredictor
-from ft.onto.base_ontology import Sentence, Token
+from forte.utils import get_class
+from forte.datasets.conll.conll_utils import write_tokens_to_file
 
 
 class CoNLLNEREvaluator(Evaluator):
     def __init__(self):
         super().__init__()
-        self.test_component = CoNLLNERPredictor().name
         self.output_file = "tmp_eval.txt"
         self.score_file = "tmp_eval.score"
         self.scores: Dict[str, float] = {}
+        if os.path.isfile(self.output_file):
+            os.remove(self.output_file)
 
-    def consume_next(self, pred_pack: DataPack, refer_pack: DataPack):
-        pred_get_data_args = {
-            "context_type": Sentence,
-            "request": {
-                Token: {
-                    "fields": ["ner"],
-                },
-                Sentence: [],  # span by default
-            },
-        }
+    def initialize(self, resources: Resources, configs: Config):
+        # pylint: disable=attribute-defined-outside-init,unused-argument
+        r"""Initialize the evaluator with `resources` and `configs`.
+        This method is called by the pipeline during the initialization.
+        Args:
+            resources (Resources): An object of class
+                :class:`forte.common.Resources` that holds references to
```

Comment:
```suggestion
                :class:`~forte.common.Resources` that holds references to
```
Adding ~ will shorten the documentation